### PR TITLE
Redesign pegin create page

### PR DESCRIPTION
**Pull request by lserra-iov**
Screenshot 2024-05-14 at 13 21 21
Screenshot 2024-05-14 at 16 13 56

### CODE REVIEW

File: src/common/plugins/vuetify.ts [hunk 1]
```diff
@@ -51,11 +51,13 @@ export const vuetify = createVuetify({
         variables: {
           'theme-on-purple': '#000',
           'theme-on-green': '#000',
+          'theme-on-pink': '#000',
```
Comment (ronaldsg20) [inline src/common/plugins/vuetify.ts:54]: why do you set this variable?

**Review by ronaldsg20 — APPROVED** [review]

File: src/common/plugins/vuetify.ts (hunk 1 above)
Comment (lserra-iov) [reply src/common/plugins/vuetify.ts:54]: By design it looks like in dark mode we should always use black over a pink background.

**Review by alexjavabraz — APPROVED** [review]
LGTM

**Review by annipi — APPROVED** [review]
LGTM!

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Pull request merged**